port: /dev/dashgo
baud: 115200
timeout: 0.1
rate: 50
sensorstate_rate: 10

use_base_controller: True
base_controller_rate: 10

# For a robot that uses base_footprint, change base_frame to base_footprint
#base_frame: base_link
base_frame: base_footprint_1

# === Robot drivetrain parameters
#wheel_diameter: 0.1212
wheel_diameter: 0.125
#wheel_diameter: 0.125
wheel_track: 0.335
encoder_resolution: 1200 # from Pololu for 13*34*4 motors
gear_reduction: 1.0
motors_reversed: False

# === PID parameters
Kp: 50
Kd: 20
Ki: 0
Ko: 50
accel_limit: 1.0
